### DISCUSSION TITLE
Deployment Guide

### DISCUSSION THREAD

**Discussion by zulfiqarpk92** [uasoft-indonesia/badaso#403] [Q&A] [answered]
Is there any guide for deploying this on server, especially shared hosting? 

I have tried building the app using `yarn build` but the public\js\badaso.js is very large and it is not loaded on the server when deployed. 

Hosted URL: https://tpos.rayyantraders.com/badaso-dashboard

Thanks

**Comment by rizkiheryandi** [comment, marked as answer]
Helo @zulfiqarpk92 , sorry for late response, forget to check the "discussion" tab

You can just deploy badaso like other common laravel application 

Did you ever deploy laravel before ?

Maybe you can learn how to deploy laravel first here https://laravel.com/docs/8.x/deployment

Thank you

**Reply by zulfiqarpk92** [reply]
@rizkiheryandi Yes, I have deployed Laravel before. But the problem is when I build the javascript bundle using the `yarn build` command it generate very large bundle file.

Thanks
